qiskit/synthesis/clifford/__init__.py

```diff
@@ -1,6 +1,6 @@
 # This code is part of Qiskit.
 #
-# (C) Copyright IBM 2017, 2018.
+# (C) Copyright IBM 2017 - 2023.
```

Comment:
I believe these should be separated by comma, i.e. "2017, 2023".

Reply:
there are several years here - so I'll list them.